docs/src/pages/premium-themes/onepirate/modules/views/ProductCategories.js

```diff
@@ -149,14 +121,32 @@ function ProductCategories(props) {
               width: image.width,
             }}
           >
-            <div
-              className={classes.imageSrc}
-              style={{
+            <Box
+              sx={{
+                position: 'absolute',
+                left: 0,
+                right: 0,
+                top: 0,
+                bottom: 0,
+                backgroundSize: 'cover',
+                backgroundPosition: 'center 40%',
                 backgroundImage: `url(${image.url})`,
               }}
             />
             <div className={classes.imageBackdrop} />
```

Comment:
I didn't know how to migrate this to emotion. Look in the documentation for the emotion and try this

```jsx
const ImageBackdrop = styled('div')(({ theme }) => ({
  position: 'absolute',
  left: 0,
  right: 0,
  top: 0,
  bottom: 0,
  background: theme.palette.common.black,
  opacity: 0.5,
  transition: theme.transitions.create('opacity'),
}))

const ImageIconButton = (MuiButtonBase)(({ theme }) => ({
  position: 'relative',
  display: 'block',
  padding: 0,
  borderRadius: 0,
  height: '40vh',
  [theme.breakpoints.down('md')]: {
    width: '100% !important',
    height: 100,
  },
  `&:hover ${ImageBackdrop}`: {
    opacity: 0.15,
  }
}))
```

 but it didn't work. Is there any other way to achieve this?

Reply:
Its usage is not so obvious, see: https://codesandbox.io/s/intelligent-swirles-mhj89?file=/src/App.js. I have pushed a new commit with an alternative.